cli/src/main/java/com/devonfw/tools/ide/context/IdeContext.java

```diff
@@ -50,12 +52,19 @@ public interface IdeContext extends IdeStartContext {
   String FOLDER_CONF = "conf";
 
   /**
-   * The base folder name of the IDE inside IDE_ROOT. Intentionally starting with an underscore and not a dot (to prevent effects like OS hiding, maven
-   * filtering, .gitignore, etc.).
+   * The nane of the folder inside IDE_ROOT reserved for IDEasy. Intentionally starting with an underscore and not a dot to prevent effects like OS hiding,
```

Comment:
```suggestion
   * The name of the folder inside IDE_ROOT reserved for IDEasy. Intentionally starting with an underscore and not a dot to prevent effects like OS hiding,
```